index.html

```diff
@@ -0,0 +1,396 @@
+                <div class="comment">A remark to what happening at 04:12 is: after clicking "Propose file change", it now goes to commit page instead of the pull request screen as shown on the video. One need to click the commit name (e.g. patch-1 if it is the first commit of your branch) near the top of the screen to go to the pull request </div>
+                <div class="reactions">REPLY</div>
+                </div> 
+              </div>
+
+              <div class="comment-section-others">
+                <div class="other-user-image">
+                  <img class="image-class" src="user-images/unnamed (1).jpg" alt="">
+                </div>
+                <div class="comment">
+                  <div class="user-name">Christine Fredricksen <span class="comment-age">2 years ago (edited)</span></div>
+                  <div class="comment">Thanks so much ! This video was really helpful and to the point.</div>
+                  <div class="reactions">REPLY</div>
+                  </div> 
+                </div>
+
+                <div class="comment-section-others">
+                  <div class="other-user-image">
+                    <img class="image-class" src="user-images/unnamed (2).jpg" alt="">
+                  </div>
+                  <div class="comment">
+                    <div class="user-name">andrewscheuermann05 <span class="comment-age">8 years ago</span></div>
+                    <div class="comment">
+                      FYI: the edit button has changed to a diagonal pencil.
+                      </div>
+                    <div class="reactions">REPLY</div>
+                    </div> 
+                  </div>
+
+                  <div class="comment-section-others">
+                  <div class="other-user-image">
+                    <img class="image-class" src="user-images/unnamed (3).jpg" alt="">
+                  </div>
+                  <div class="comment">
+                    <div class="user-name">Sampath Lokuge <span class="comment-age">6 years ago</span></div>
+                    <div class="comment">
+                      Nice.Thanks a lot :)
+                      </div>
+                    <div class="reactions">REPLY</div>
+                    </div> 
+                  </div>
+        </section>
+      </div>
+      <div class="section2">
+       <div class="ad-space">
+        ads video
+       </div>
+
+<div class="classification-buttons">
+  <button class="current-button">
+    All
+  </button>
+  <button class="category-button">
+    Source code
+  </button>
+  <button class="category-button">
+    Listenable
+  </button>
+  <button class="category-button">
+    Related
+  </button>
+</div>
+
+       <div class="video-thumbnails">
+        <div class="thumbnail">
+          <div class="thumbnail-container">
+            <div class="time-duration">
+              14:25
+            </div>
+            <img class="thumbnail-image" src="thumbnails/thumbnail(1).webp" alt="upcoming">
```

Comment:
Kindly add a relevant alternative (alt="relevant name to the image")name to the img element

Reply:
- [ ]  Kindly phrase your comments in a polite and considerate manner, like a suggestion, instead of a command. This will make it easy for the student to accept and implement them. :wink: